compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -305,7 +308,11 @@ void addPackPeelBasedPassPipeline(OpPassManager &funcPassManager,
   // Lower to UKernels.
   {
     AMDAIELowerToUKernelsOptions options;
-    options.pathToUkernels = clPathToUkernels;
+    std::filesystem::path pathToUkernels = clPathToUkernels.getValue();
+    // quote for windows paths like `C:/Program Files`
+    std::stringstream ss;
+    ss << pathToUkernels.make_preferred();
+    ss >> std::quoted(options.pathToUkernels);
```

Comment:
Plenty of ways to do this. Here's some other code to reference:
* https://github.com/iree-org/iree/blob/f224bc88652ee002131ab863490233e4e5e78e37/compiler/src/iree/compiler/Utils/ToolUtils.h#L17-L24
* https://github.com/iree-org/iree/blob/f224bc88652ee002131ab863490233e4e5e78e37/compiler/src/iree/compiler/Utils/ToolUtils.cpp#L24-L39

(You might even be able to use those APIs from IREE here, instead of writing your own, or use similar APIs from LLVM)

Reply:
> Does LLVM have a utility for generating UUIDs?

Does it? Looks like lldb (lld?) does but it's not in Support or anything https://github.com/search?q=repo%3Allvm%2Fllvm-project%20uuid&type=code

> Here's some other code to reference:

cool